code/modules/xenoarchaeology/traits/traits.dm

```diff
@@ -0,0 +1,260 @@
+/*
+	For laternate versions of traits, that you want as seperate on the labeler, use the format-
+	'trait_name Δ' 'trait_name Σ' 'trait_name Ω', for up to three alternates, add more symbols if you have a trait with 4 alts or more
+*/
+
+/datum/xenoartifact_trait
+	///Reference to the artifact
+	var/datum/component/xenoartifact/parent
```

Comment:
this should be a different name like `parent_comp`, `xeno_comp`, `parent_xno_comp`, or something else.
I'd like to this be `parent_xeno_comp` since it can tell its parent of the system, and is a (xeno) component.